src/controllers/UserAjax.php

```diff
@@ -297,10 +297,10 @@ public function getGetusers()
 
     }
 
-    public function getGetUser()
```

Comment:
make sure you merge in the latest updates from the current `develop` branch. I think a change I made last week may get clobbered by this PR